packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -167,7 +167,10 @@ public static void setCustomClientBuilder(CustomClientBuilder ccb) {
     customClientBuilder = ccb;
   }
 
-  public static interface CustomClientBuilder {
+  /**
+   * @deprecated To be removed in a future release. See https://github.com/facebook/react-native/pull/37798#pullrequestreview-1518338914
+   */
+  public static interface CustomClientBuilder extends com.facebook.react.modules.network.CustomClientBuilder {
     public void apply(OkHttpClient.Builder builder);
```

Comment:
This can just be an empty interface.

Reply:
Implemented in latest commit; resolve if sufficient.